plugins/sinks/file/file.go

```diff
@@ -60,7 +62,16 @@ func (s *Sink) Init(ctx context.Context, config map[string]interface{}) (err err
 	if err := s.validateFilePath(s.config.Path); err != nil {
```

Comment:
do we still need to do this? inferring format from the extension.

it looks like our `format` config is not working cause of this.

Reply:
hey, more context on what is not working??
I basically use this function to ensure that filename is valid, like it has extension. I am not inferring the format from this function.

Reply:
I am trying with this config
```
      path: postgres_food_app_data.xxx
      format: ndjson
```
but the output is in yaml

Reply:
hey mas, can you check again, It is working for me.